widget/embedded/src/store/slices/data.ts

```diff
@@ -95,13 +98,47 @@ export const createDataSlice: StateCreator<
     const blockchains = get().blockchains({
       type: options.type,
     });
+    const includedTokens = new Set<string>();
+    const excludedTokens = new Set<string>();
+    if (supportedTokensConfig) {
+      const shouldIncludeOrExcludeTokens = (
```

Comment:
I think `shouldIncludeOrExcludeTokens` function doesn't give a good idea of ​​what it's supposed to do.